src/App.tsx

```diff
@@ -150,6 +150,21 @@ function App() {
         tokenId="0"
         preset="original"
       /> */}
+
+      {/* token with html asset type */}
+      <Asset
```

Comment:
this part is not commented, is it intentional?

Reply:
yes App file is just for test